.gitignore

```diff
@@ -21,6 +21,9 @@ __pycache__
 /*_data.json
 /*_embeddings.bin
 
+# pytest configuration
+pytest.ini
+
```

Comment:
(My comment seemed lost, or I am confused by GitHub's UI.)

Why do we need this? I'd imagine *if* we had a pytest.ini file we'd want it in git.